_data/sidebars/lb3_sidebar.yml

```diff
@@ -899,10 +907,6 @@ children:
       url: Storage-connector.html
       output: 'web, pdf'
 
-    - title: 'Swagger connector'
```

Comment:
I'm thinking whether we should keep this rather than removing this entry.  The `loopback-connector-swagger` has been deprecated but probably there're still some users using it.
Either way is fine with me, but I'd like to hear thoughts from @raymondfeng.

Reply:
I was following the [issue description](https://github.com/strongloop/loopback-next/issues/3714) but I'll wait for @raymondfeng's thoughts.

Reply:
I see. It's probably updated while discussing this issue during the estimation meeting.

Reply:
For LB3, we should leave `Swagger connector` there as-is.
For LB4, we should only list `OpenAPI connector`.